src/moderation/moderation.service.ts

```diff
@@ -31,7 +31,7 @@ export class ModerationService {
     async filterUsersForPost(
         userIds: number[],
         post: Post,
-        repostedBy?: number,
+        interactionAccountId?: number,
```

Comment:
Changed this to `interactionAccountId` so the same logic can be used for `like` as well as `repost` (in both cases we want the same thing to happen - prevent feed insertion and prevent notification)